src/Stratis.Bitcoin/Utilities/FileStorage.cs

```diff
@@ -78,6 +78,10 @@ public void SaveToFile(T toSave, string fileName, bool saveBackupFile = false)
                 // Marking the file for deletion in the future.
                 File.Move(tempFilePath, $"{ filePath}.{ uniqueId}.del");
             }
+            catch (UnauthorizedAccessException)
```

Comment:
You don't want to swallow the exception here.

Reply:
Fixed